paper-button-behavior.html

```diff
@@ -42,7 +60,44 @@
       } else if (this.receivedFocusFromKeyboard) {
         e = 3;
       }
-      this._elevation = e;
+      this.elevation = e;
+    },
+
+    _computeKeyboardClass: function(receivedFocusFromKeyboard) {
+      this.classList.toggle('keyboard-focus', receivedFocusFromKeyboard);
+    },
+
+    _downAction: function(event) {
+      if (!this._ripple) {
+        this._makeRipple(event);
+      }
+    },
+
+    _receivedInitialFocus: function(focused) {
```

Comment:
The naming here suggests to me that `_receivedInitialFocus` will only be called once, when `focused` becomes true, but this is not the case. Just observing.